rt_utils/rtstruct_builder.py

```diff
@@ -13,14 +13,14 @@ class RTStructBuilder:
     """
 
     @staticmethod
-    def create_new(dicom_series_path: str) -> RTStruct:
+    def create_new(dicom_series_path: str, warn_only: bool = False) -> RTStruct:
```

Comment:
Do we need this in the new method? Looks like its only used during existing rt struct validation

Reply:
Indeed, I refactored it so it is only passed to create_new, then down to validate_rtstruct_series_references and validate_contour_image_in_series_data